src/main/java/net/torocraft/minecoprocessors/processor/Processor.java

```diff
@@ -565,7 +573,7 @@ private void testProcessCall() {
       assert ip == (short) 0xabcd;
       assert sp == 0;
 
-    } catch (Exception e) {
+    } catch (ParseException e) {
```

Comment:
Why do you only want to fail the tests for `ParseException`?

Reply:
Any other kind of exception will bubble up and fail them anyway. The only reason we need to even catch this one is that it's a checked exception.